api/command_leave_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
update here to follow the standard

`// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.`